vision/classification/shufflenet_v2/shufflenet-export.py

```diff
@@ -0,0 +1,119 @@
+import torch
+import onnxruntime
+from onnx import numpy_helper
+from PIL import Image
+from torchvision import transforms
+
+import numpy as np
+import os
+import urllib
+
+#           GitHub Repo    |    Model   
+MODELS = [
+    ('pytorch/vision:v0.5.0', 'shufflenet_v2_x0_5'),
+    ('pytorch/vision:v0.5.0', 'shufflenet_v2_x1_0'),
+]
+data_dir = 'test_data_set_0'
+
+url, filename = ("https://github.com/pytorch/hub/raw/master/dog.jpg", "dog.jpg")
+urllib.URLopener().retrieve(url, filename)
+
+
+def save_tensor_proto(file_path, name, data):
+    tp = numpy_helper.from_array(data)
+    tp.name = name
+
+    with open(file_path, 'wb') as f:
+        f.write(tp.SerializeToString())
+
+
+def save_data(test_data_dir, prefix, data_list):
+    if isinstance(data_list, torch.autograd.Variable) or isinstance(data_list, torch.Tensor):
+        data_list = [data_list]
+    for i, d in enumerate(data_list):
+        d = d.data.cpu().numpy()
+        save_tensor_proto(os.path.join(test_data_dir, '{0}_{1}.pb'.format(prefix, i)), prefix + str(i+1), d)
+
+
+def save(name, model, inputs, outputs, input_names=['input'], output_names=['output'], **kwargs):
+    if hasattr(model, 'train'):
+        model.train(False)
+    dir = './'
+    if not os.path.exists(dir):
+        os.makedirs(dir)
+    dir = os.path.join(dir, 'test_' + name)
+    if not os.path.exists(dir):
+        os.makedirs(dir)
+
+    def f(t):
```

Comment:
I still feel that names f and g could be more specific to clarify for new users -- unless you have a reason for naming the helper functions this way?